styles.css

```diff
@@ -87,7 +87,7 @@
 
 .xp {
   position: static;
-  font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] Kindly do the same for Poppins as well, [link](https://fonts.google.com/specimen/Poppins)